config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216604559-3a6e6f52-d254-423b-ad90-758c3d4fb704.png)

- [ ] Kindly remove your name from here because you have your name here two times which is making your readme file not professional :thinking: kindly remove this one which has been shown in the image above :arrow_up: